base/threadcall.jl

```diff
@@ -1,7 +1,7 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
 const max_ccall_threads = parse(Int, get(ENV, "UV_THREADPOOL_SIZE", "4"))
-const thread_notifiers = Union{Base.Condition, Nothing}[nothing for i in 1:max_ccall_threads]
+const thread_notifiers = Union{Threads.Condition, Nothing}[nothing for i in 1:max_ccall_threads]
```

Comment:
Do we want this to be `ThreadSynchronizer` (e.g. SpinLock backed Condition)?

Reply:
That's the plan. More precisely a Base.Event which now uses a spinlock